packages/tools/kadena-cli/src/account/commands/accountAddManual.ts

```diff
@@ -31,6 +36,19 @@ export const createAddAccountManualCommand = createCommand(
 
     const chainId = (await option.chainId()).chainId;
 
+    // when --quiet is passed
+    // network is not provided and
+    // default network is also not found
+    if (!isNotEmptyString(network)) {
```

Comment:
if `--quiet` is passed, it will throw an error only if the flag is not passed AND the option is required. So in this case I assume `network` is not required, and I see you repeating this check in several places.

I'm not sure how complex it would be, but have you looked into making network required, and solving this once at the place of the `quiet` checks?

Reply:
Yeah, I agree even I'm not happy with current approach with multiple duplication of doing it all place. 

Yes, I tried out introducing default but I was getting same error of missing required arguments initially. Maybe I'll try out something tomorrow morning first thing. Thank you for the feedback